### PR TITLE
fix ds4 alignment test

### PR DESCRIPTION
**Pull request by msendlakowski**
## Description
Updated HTML on test to match new skin requirements
Fixes https://github.com/eBay/skin/issues/285

## Context
Skin updated

## Screenshots
screen shot 2018-09-11 at 2 19 47 pm

### CODE REVIEW

File: docs/test/misc/ds4/alignment-controls-grid.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h2>Small</h2>
                     <option value="3">Option 3</option>
                 </select>
                 <span class="select__icon" hidden>
-                    <svg aria-hidden="true" focusable="false">
+                    <svg hidden="true" focusable="false">
```
Comment (seangates) [inline docs/test/misc/ds4/alignment-controls-grid.html]: Is should just be `hidden` with no assignment of value.

File: docs/test/misc/ds4/alignment-controls-grid.html [hunk 2]
```diff
@@ -51,17 +51,15 @@ <h2>Medium</h2>
...
-                            <use xlink:href="#svg-icon-arrow-down"></use>
-                        </svg>
-                    </span>
+                <select class="select__control">
+                    <option value="1">Option 1</option>
+                    <option value="2">Option 2</option>
+                    <option value="3">Option 3</option>
+                </select>
+                <span class="select__icon" hidden>
+                    <svg hidden="true" focusable="false">
```
Comment (seangates) [inline docs/test/misc/ds4/alignment-controls-grid.html]: Same as above. `hidden`

File: docs/test/misc/ds4/alignment-controls-grid.html [hunk 3]
```diff
@@ -83,17 +81,15 @@ <h2>Large</h2>
...
```
Comment (seangates) [inline docs/test/misc/ds4/alignment-controls-grid.html]: Same as above. `hidden`

File: docs/test/misc/ds4/alignment-controls-grid.html (hunk 1 above)
Comment (msendlakowski) [reply docs/test/misc/ds4/alignment-controls-grid.html]: Fixed this by removing the unneeded svg completely

File: docs/test/misc/ds4/alignment-controls-grid.html (hunk 2 above)
Comment (msendlakowski) [reply docs/test/misc/ds4/alignment-controls-grid.html]: Fixed this by removing the unneeded svg completely

File: docs/test/misc/ds4/alignment-controls-grid.html (hunk 3 above)
Comment (msendlakowski) [reply docs/test/misc/ds4/alignment-controls-grid.html]: Fixed this by removing the unneeded svg completely

**Pull request merged**